lib/src/date.dart

```diff
@@ -52,6 +52,13 @@ abstract class Date implements Comparable<Date> {
   /// Milliseconds [0..999]
   int get millisecond;
 
+  /// Returns the time part in milliseconds.
+  int get _timeInMilliseconds =>
+      (hour * 60 * 60 * 1000) +
```

Comment:
First, please use block notation for methods instead of experission notation.

Second, instead of time in milliseconds,
Please add a public method named `time` which returns time part in `Duration`. Then use that duration's compare to method if is needed.

Is this ok ?

Reply:
Then you can access time in millisecond by using `time.inMilliseconds` from outside ...

Reply:
And it has some benefits for the methods which will be added in next release,
Like addTime and etc